raiden/tests/utils/smoketest.py

```diff
@@ -168,7 +164,10 @@ def load_or_create_smoketest_config():
         with open(smoketest_config_path) as handler:
             smoketest_config = json.load(handler)
         # if the file versions still fit, return the genesis config (ignore solc if not available)
-        if all(versions[key] == smoketest_config['versions'][key] for key in versions.keys()):
+        if all(
```

Comment:
please, make don't use multiline ifs, move the condition up and give a it a nice name.